src/activespeakerpolicy/DefaultActiveSpeakerPolicy.ts

```diff
@@ -1,11 +1,44 @@
-// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// Copyright 2019-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
 import ActiveSpeakerPolicy from './ActiveSpeakerPolicy';
 
 export default class DefaultActiveSpeakerPolicy implements ActiveSpeakerPolicy {
+  /**
+   * Map of attendeeIds to their active speaker score values
+   */
   private volumes: { [attendeeId: string]: number } = {};
 
+  /** Creates active speaker policy with speakerWeight, cutoffThreshold, silenceThreshold, and takeoverRate.
+   *
+   * @param speakerWeight
+   * Number used to calculate new active speaker score for current attendee
```

Comment:
optional: start with an article in four fields, e.g. `Number` => `The number`  or `A number`

Reply:
Changed for the fields, used `The`. Thanks!